afteraction-share.html

```diff
@@ -409,6 +406,22 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 							}
 						}
 					}
+
+					// Set Whatsapp Share URL based on response from API
+					var getShareURL = '/share/link?type=ot&page_name={{page.name}}&short=1';
+					$.ajax({
+						method: 'GET',
+						url: getShareURL,
+					}).done(function(shareURL) {
```

Comment:
It looks like this is called once a user submits a form. If the API takes a long time or doesn't respond, what happens?

Reply:
Also, would this trigger the share tracking as in here? https://github.com/350org/action-tools/issues/49

Reply:
yes - reverted this commit. I think this is a little easier to handle because we can call the API when the user clicks "Share on Whatsapp"